grammar.js

```diff
@@ -1,5 +1,11 @@
-const DIGITS = token(sep1(/[0-9]+/, /_+/))
-const HEX_DIGITS = token(sep1(/[A-Fa-f0-9]+/, '_'))
+// deno-lint-ignore-file ban-ts-comment
```

Comment:
It would have been better to do this estethic change in another PR and keep this PR
just for what's the title says.

Reply:
ok, sure